commons/src/main/java/io/hyscale/commons/utils/HyscaleStringUtil.java

```diff
@@ -43,4 +43,18 @@ public static String removeSuffixChar(StringBuilder input, char trailingChar) {
         return removeSuffixStr(input.toString(), Character.toString(trailingChar));
     }
 
+    public static final String stringArrayToString(String[] strArray, String delimiter) {
```

Comment:
This can be removed

Reply:
This is used to create custom string from array with provided delimiter